### PR TITLE
Implement Global::get/set for refs

### PR DESCRIPTION
**Pull request by rossberg**
Follow-up to #93, implementing missing functionality to also allow accessing globals of ref type directly through API.

Also includes bugfix in C API's internal borrowed_val.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ OUT_DIR = out
 
 # Example config
 EXAMPLE_OUT = ${OUT_DIR}/${EXAMPLE_DIR}
-EXAMPLES = hello callback trap reflect global table memory finalize serialize threads hostref
+EXAMPLES = hostref #hello callback trap reflect global table memory finalize serialize threads hostref
```
Comment (jakobkummerow) [inline Makefile]: You probably want to drop the `#` here

File: src/wasm-v8.cc [hunk 2]
```diff
@@ -1154,6 +1154,7 @@ template<> struct implement<Ref> { using type = RefImpl<Ref>; };
 
 
 Ref::~Ref() {
+printf("deleting ref\n");
```
Comment (jakobkummerow) [inline src/wasm-v8.cc]: debugging leftover?

File: Makefile (hunk 1 above)
Comment (rossberg) [reply Makefile]: Done.

File: src/wasm-v8.cc (hunk 2 above)
Comment (rossberg) [reply src/wasm-v8.cc]: Removed.

**Pull request merged**